helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -82,7 +82,10 @@ export class CheckChatStatus {
 					const isChatAccepted = checkForEvent(messageArray, 'ChatEstablished');
 					if (isChatAccepted === true) {
 						//TODO: Add queue_time to analytics
-						this.modify.getAnalytics().sendEvent(getEventData(this.data.room.id, EventName.ESCALATION_SUCCESSFUL, { queue_time: '' }));
+						const chatKey = this.key.split('!')[0];
```

Comment:
Why do we need to split here `this.key.split('!')[0]`? 

In the PR https://github.com/WideChat/Apps.Salesforce.LiveAgent/pull/68 `ChatKey` was `id` from the response of `System/SessionId` Endpoint but here `ChatKey` is `key`?

Reply:
@csmith14 That makes sense. Thanks for the clarification 👍